saleor/tests/e2e/orders/test_staff_can_create_order_from_checkout.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+from ..checkout.utils import checkout_create, checkout_delivery_method_update
+from ..product.utils.preparing_product import prepare_product
+from ..shop.utils.preparing_shop import prepare_shop
+from ..utils import assign_permissions
+from .utils import order_create_from_checkout
+
+
+@pytest.mark.e2e
+def test_staff_can_create_order_from_checkout_CORE_0215(
```

Comment:
this case should be `test_app_can_careate_order_from_checkout` as this mutation is dedicated to the checkout app :)